meta-xt-control-domain-virtio/recipes-guest/doma/files/virtio-env-no-weston.conf

```diff
@@ -0,0 +1,3 @@
+[Unit]
+Requires=backend-ready@bridge.service
```

Comment:
Are you sure you need this dependency? I believe doma can start right away (i.e. without any additional dependencies) if there is no wayland.

Reply:
Removed